src/rating/Rating.js

```diff
@@ -245,6 +293,23 @@ const styles = StyleSheet.create({
   },
 });
 
+const fractionsType = (props, propName, componentName) => {
+  if (props[propName]) {
+    const value = props[propName];
+    if (typeof value === 'number') {
+      return value >= 0 && value <= 20
+        ? null
+        : new Error(
+            `\`${propName}\` in \`${componentName}\` must be between 0 and 20`
```

Comment:
this is a great addition to validate if the fractions is between 0 and 20. 

Although seems like an overkill to me. Is this absolutely necessary? Can't the user get away with not passing a `fractions` prop if they don't need fractions?